src/80850_len_3060.c

```diff
@@ -803,278 +803,300 @@ void update_status_menu(void) {
+        hud_element_set_render_pos(id, x, y);
         hud_element_draw_next(id);
 
         id = uiStatus->coinSparkleIconIndex;
-        hud_element_set_render_pos(id, uiStatus->drawPosX + 244, uiStatus->drawPosY + 14);
+        x = uiStatus->drawPosX + 244;
+        y = uiStatus->drawPosY + 14;
+        hud_element_set_render_pos(id, x, y);
         hud_element_draw_next(id);
 
-        status_menu_draw_number(uiStatus->iconIndexB, uiStatus->drawPosX + 247, uiStatus->drawPosY + 8, uiStatus->displayCoins, 3);
+        x = uiStatus->drawPosX + 247;
+        y = uiStatus->drawPosY + 8;
+        status_menu_draw_number(uiStatus->iconIndexB, x, y, uiStatus->displayCoins, 3);
     }
 
     id = uiStatus->starIconIndex;
-    s6 = 1;
-
+    showStat = TRUE;
     if (uiStatus->spBlinking) {
         if (uiStatus->spBlinkCounter > 5) {
             if (uiStatus->spBlinkCounter <= 8) {
-                s6 = 0;
+                showStat = FALSE;
             } else {
                 uiStatus->spBlinkCounter = 0;
-                s6 = 1;
+                showStat = TRUE;
             }
         }
         uiStatus->spBlinkCounter++;
     }
 
-    fp = uiStatus->drawPosX + 20;
-    s5 = uiStatus->drawPosY + 28;
+    x = uiStatus->drawPosX + 20;
+    y = uiStatus->drawPosY + 28;
 
-    temp = uiStatus->displaySP / 256;
-    s4 = uiStatus->displaySP % 256;
-    s4 /= 32;
-    s4 = s4 + temp * 8;
+    spBars = uiStatus->displaySP / 256;
+    limit = uiStatus->displaySP % 256;
+    limit /= 32;
+    limit = limit + spBars * 8;
     if (uiStatus->unk_57 == 1) {
-        temp = playerData->specialBarsFilled / 256;
-        s4 = playerData->specialBarsFilled % 256;
-        s4 = s4 / 32;
-        s4 += temp * 8;
-        func_800F0CB0(0, fp + s4 * 25 / 10, s5, 1.0f);
+        spBars = playerData->specialBarsFilled / 256;
+        limit = playerData->specialBarsFilled % 256;
+        limit = limit / 32;
+        limit += spBars * 8;
+        func_800F0CB0(0, x + limit * 25 / 10, y, 1.0f);
         uiStatus->unk_57 = 2;
     }
 
-    sp54 = 0;
-    new_var = fp;
+    sp54 = FALSE;
     if (uiStatus->unk_57 != 0) {
         if (uiStatus->unk_58 != 0) {
             uiStatus->unk_58--;
         } else {
             uiStatus->unk_57 = 0;
         }
         if ((uiStatus->unk_58 / 5) & 1) {
-            sp54 = 1;
+            sp54 = TRUE;
         }
+        maxStarPower = uiStatus->unk_59; // required to match
         s7 = uiStatus->unk_59 % 8;
         s7 += uiStatus->unk_59 / 8 * 8;
     } else {
-        s7 = s4;
+        s7 = limit;
     }
 
-    s2 = 0;
+    i = 0;
     sp50 = 0;
     s1 = 0;
 
     if (uiStatus->spBlinking) {
-        if (!s6) {
-            if (sp50 < uiStatus->spBarsToBlink * 8) {
+        if (!showStat) {
+            s32 limit = uiStatus->spBarsToBlink * 8;
+            do {} while (0);
+            if (sp50 < limit) {
                 while (TRUE) {
-                    s2++;
+                    i++;
```

Comment:
I think I prefer the more concise formatting here, how does everyone else feel?

Reply:
I think in this case it works, sure. but this is the kind of thing a code formatter would undo anyway, probably, so I don't think it matters much